Source/Core/DolphinQt/Debugger/CodeDiffDialog.cpp

```diff
@@ -0,0 +1,515 @@
+
+  m_matching_results_table = new QTableWidget();
+  m_matching_results_table->setColumnCount(5);
+  m_matching_results_table->setHorizontalHeaderLabels(
+      {tr("Address"), tr("Total Hits"), tr("Hits"), tr("Symbol"), tr("Inspected")});
+  m_matching_results_table->setSelectionMode(QAbstractItemView::SingleSelection);
+  m_matching_results_table->setSelectionBehavior(QAbstractItemView::SelectRows);
+  m_matching_results_table->setSizePolicy(QSizePolicy::Expanding, QSizePolicy::Expanding);
+  m_matching_results_table->setContextMenuPolicy(Qt::CustomContextMenu);
+  m_matching_results_table->setColumnWidth(0, 60);
+  m_matching_results_table->setColumnWidth(1, 60);
+  m_matching_results_table->setColumnWidth(2, 4);
+  m_matching_results_table->setColumnWidth(3, 210);
+  m_matching_results_table->setColumnWidth(4, 65);
+  m_reset_btn = new QPushButton(tr("Reset All"));
+  m_reset_btn->setSizePolicy(QSizePolicy::Fixed, QSizePolicy::Fixed);
+  m_help_btn = new QPushButton(tr("Help"));
+  m_help_btn->setSizePolicy(QSizePolicy::Fixed, QSizePolicy::Fixed);
+  auto* help_reset_layout = new QHBoxLayout;
+  help_reset_layout->addWidget(m_reset_btn, 0, Qt::AlignLeft);
+  help_reset_layout->addWidget(m_help_btn, 0, Qt::AlignRight);
+
+  auto* layout = new QVBoxLayout();
+  layout->addLayout(btns_layout);
+  layout->addLayout(labels_layout);
+  layout->addWidget(m_matching_results_table);
+  layout->addLayout(help_reset_layout);
+
+  setLayout(layout);
+}
+
+void CodeDiffDialog::ConnectWidgets()
+{
+  connect(m_record_btn, &QPushButton::toggled, this, &CodeDiffDialog::OnRecord);
+  connect(m_include_btn, &QPushButton::pressed, [this]() { Update(true); });
+  connect(m_exclude_btn, &QPushButton::pressed, [this]() { Update(false); });
+  connect(m_matching_results_table, &QTableWidget::itemClicked, [this]() { OnClickItem(); });
+  connect(m_reset_btn, &QPushButton::pressed, this, &CodeDiffDialog::ClearData);
+  connect(m_help_btn, &QPushButton::pressed, this, &CodeDiffDialog::InfoDisp);
+  connect(m_matching_results_table, &CodeDiffDialog::customContextMenuRequested, this,
+          &CodeDiffDialog::OnContextMenu);
+}
+
+void CodeDiffDialog::OnClickItem()
+{
+  UpdateItem();
+  auto address = m_matching_results_table->currentItem()->data(Qt::UserRole).toUInt();
+  m_code_widget->SetAddress(address, CodeViewWidget::SetAddressUpdate::WithDetailedUpdate);
+}
+
+void CodeDiffDialog::ClearData()
+{
+  if (m_record_btn->isChecked())
+    m_record_btn->toggle();
+  ClearBlockCache();
+  m_matching_results_table->clear();
+  m_matching_results_table->setRowCount(0);
+  m_matching_results_table->setHorizontalHeaderLabels(
+      {tr("Address"), tr("Total Hits"), tr("Hits"), tr("Symbol"), tr("Inspected")});
+  m_matching_results_table->setEditTriggers(QAbstractItemView::EditTrigger::NoEditTriggers);
+  m_exclude_size_label->setText(QStringLiteral("Excluded: 0"));
+  m_include_size_label->setText(QStringLiteral("Included: 0"));

```

Comment:
These need to be `tr()`. Maybe also use the same tr("Excluded: %1").arg(0) here so it only shows up once in the translation files.